src/Notifier.php

```diff
@@ -23,9 +23,10 @@ class Notifier
     public function __construct($opt = [])
     {
         // TODO: test that projectId and projectKey exists
-        $this->opt = array_merge($opt, [
+        $this->opt = array_merge([
             'host' => 'api.airbrake.io',
-        ]);
+            'secure' => true,
```

Comment:
You are right that we had such option in old Airbrake Ruby notifier, but we don't have it in new notifiers since it is redundant.